### PR TITLE
Merge lukasgabriel/Ciphey/master

### PR DESCRIPTION
**Pull request by lukasgabriel**
Add atbash decoding and corresponding test.

### CODE REVIEW

**Comment by brandonskerritt** [comment]
@all-contributors please add @lukasgabriel for code

File: ciphey/basemods/Decoders/atbash.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    def __init__(self, config: Config):
+        super().__init__(config)
+
+    @staticmethod
+    def getParams() -> Optional[Dict[str, ParamSpec]]:
+        return None
+
+    @staticmethod
+    def getTarget() -> str:
+        return 'atbash'
```
Comment (brandonskerritt) [inline ciphey/basemods/Decoders/atbash.py:50]: Python Black will add a newline to the end of the file

File: tests/test_main.py [hunk 2]
```diff
@@ -75,6 +75,9 @@ def test_hex():
 
     assert res.lower() == answer_str.lower()
 
+def test_atbash():
```
Comment (brandonskerritt) [inline tests/test_main.py:78]: Run Python black on this file too :)

File: ciphey/basemods/Decoders/atbash.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
+# atbash.py
+# by lukasgabriel
```
Comment (brandonskerritt) [inline ciphey/basemods/Decoders/atbash.py:2]: You can use a GitHub profile link here instead if you like :)

File: ciphey/basemods/Decoders/atbash.py [hunk 4]
```diff
@@ -0,0 +1,50 @@
+# atbash.py
```
Comment (brandonskerritt) [inline ciphey/basemods/Decoders/atbash.py:1]: The name of this file isn't needed as a comment, as the file name is atbash.py :)

Can you change this to:

```
# community
```

to denote this is a community module? will be useful for tagging system #147 :sparkles:

File: ciphey/basemods/Decoders/atbash.py (hunk 3 above)
Comment (brandonskerritt) [reply ciphey/basemods/Decoders/atbash.py:2]: Actually, I'd prefer it if you added this.

File: ciphey/basemods/Decoders/atbash.py [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+# by lukasgabriel
+
+from typing import Optional, Dict, List
+
+from ciphey.iface import Config, ParamSpec, T, U, Decoder, registry
+
```
[the remark this answers is not in the document]
Comment (lukasgabriel) [reply ciphey/basemods/Decoders/atbash.py:7]: Yeah sure, even if I'm more of an autopep8 guy myself  😉

**Pull request merged**